aws-lc-sys-testing/build.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+fn main() {
+    // ensure that the include path is exported and set up correctly
+    cc::Build::new()
+        .include(env("DEP_AWS_LC_0_12_0_INCLUDE"))
+        .file("src/testing.c")
+        .compile("aws_ls_sys_testing");
+
+    // ensure the libcrypto artifact is linked
+    println!("cargo:rustc-link-lib=aws_lc_0_12_0_crypto");
```

Comment:
I'd say just to reduce the number of times we need to track this version number add a build dependency on toml 0.8.

```suggestion
let content = read_to_string("../aws-lc-sys/Cargo.toml").unwrap();

    let table = content.parse::<Table>().unwrap();

    let name = table["package"]
        .as_table()
        .unwrap()
        .get("links")
        .unwrap()
        .as_str()
        .unwrap();

    // ensure that the include path is exported and set up correctly
    cc::Build::new()
        .include(env(format!("DEP_{}_INCLUDE", name.to_uppercase())))
        .file("src/testing.c")
        .compile("aws_ls_sys_testing");

    // ensure the libcrypto artifact is linked
    println!("cargo:rustc-link-lib={}_crypto", name);
```

Reply:
Fixed